environments/Test-Laptop.json

```diff
@@ -63,7 +41,6 @@
        "servers" : [ "0.pool.ntp.org", "1.pool.ntp.org", "2.pool.ntp.org", "3.pool.ntp.org" ]
     },
     "chef_client": {
-      "server_url": "http://10.0.100.3:4000",
```

Comment:
I assume the OpenStack code will reside in a separate branch in chef-bach? If not how does one handle merging the environment file with the master version?